element_deeplabcut/model.py

```diff
@@ -683,28 +693,41 @@ def make(self, key):
         # ID model and directories
         dlc_model = (Model & key).fetch1()
 
-        task_mode, analyze_video_params, output_dir = (PoseEstimationTask & key).fetch1(
-            "task_mode", "pose_estimation_params", "pose_estimation_output_dir"
+        assert dlc_model["project_path"], (
+            "Your model table must have a 'project_path'"
+            + "field pointing to a DLC directory"
         )
-        analyze_video_params = analyze_video_params or {}
-        output_dir = find_full_path(get_dlc_root_data_dir(), output_dir)
-        video_filepaths = [
-            find_full_path(get_dlc_root_data_dir(), fp).as_posix()
-            for fp in (VideoRecording.File & key).fetch("file_path")
-        ]
-        project_path = find_full_path(
-            get_dlc_root_data_dir(), dlc_model["project_path"]
+        task_mode, output_dir = (PoseEstimationTask & key).fetch1(
+            "task_mode", "pose_estimation_output_dir"
         )
 
+        output_dir = find_full_path(get_dlc_root_data_dir(), output_dir)
+
         # Triger PoseEstimation
         if task_mode == "trigger":
+            # Triggering dlc for pose estimation required:
+            # - project_path: full path to the directory containing the trained model
+            # - video_filepaths: full paths to the video files for inference
+            # - analyze_video_params: optional parameters to analyze video
+            project_path = find_full_path(
+                get_dlc_root_data_dir(), dlc_model["project_path"]
+            )
+            video_filepaths = [
+                find_full_path(get_dlc_root_data_dir(), fp).as_posix()
+                for fp in (VideoRecording.File & key).fetch("file_path")
+            ]
+            analyze_video_params = (PoseEstimationTask & key).fetch1(
+                "pose_estimation_params"
+            ) or {}

```

Comment:
I think this `fetch1` would throw an error if it didn't get exactly one entry
```suggestion
            analyze_video_params = (PoseEstimationTask & key).fetch(
                "pose_estimation_params", limit=1
            )[0] or {}

```

Reply:
How so? There must be only one unique entry in `PoseEstimationTask` associated with this `key`

Reply:
My mistake - The `fetch1` here would return the `null` stored under `pose_estimation_params`